packages/pds-ember/package.json

```diff
@@ -2,21 +2,89 @@
   "name": "@hashicorp/pds-ember",
   "version": "0.0.10",
   "description": "Ember addon for building Structure-styled UIs",
-  "main": "index.js",
-  "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
-  },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/hashicorp/structure.git"
-  },
   "keywords": [
     "ember-addon"
   ],
-  "author": "Ryan Johnson",
   "license": "MPL-2.0",
+  "author": "Ryan Johnson",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/hashicorp/structure.git"
+  },
   "bugs": {
     "url": "https://github.com/hashicorp/structure/issues"
   },
-  "homepage": "https://github.com/hashicorp/structure#readme"
+  "homepage": "https://github.com/hashicorp/structure#readme",
+  "directories": {
+    "doc": "doc",
+    "test": "tests"
+  },
+  "scripts": {
+    "build": "ember build --environment=production",
+    "lint:hbs": "ember-template-lint .",
+    "lint:js": "eslint .",
+    "start": "ember serve",
+    "test": "npm-run-all lint:* test:*",
+    "test:ember": "ember test",
+    "test:ember-compatibility": "ember try:each",
+    "lint-all": "yarn lint:hbs && yarn lint:js",
+    "g:component": "ember g pds-component",
+    "d:component": "ember d pds-component",
+    "storybook": "start-storybook -p 6006 -s dist",
+    "storybook:build": "build-storybook"
+  },
+  "dependencies": {
+    "broccoli-funnel": "^3.0.3",
+    "broccoli-merge-trees": "^4.2.0",
+    "ember-cli-babel": "^7.22.1",
+    "ember-cli-htmlbars": "^5.3.1",
+    "ember-cli-sass": "^10.0.1",
+    "ember-cli-string-utils": "^1.1.0",
+    "ember-svg-jar": "^2.2.3",
+    "sass": "^1.26.11"
+  },
+  "devDependencies": {
+    "@ember/optional-features": "^2.0.0",
+    "@glimmer/component": "^1.0.1",
+    "@glimmer/tracking": "^1.0.1",
+    "@hashicorp/structure-icons": "^1.8.1",
+    "@storybook/addon-docs": "^6.0.22",
+    "@storybook/addon-knobs": "^6.0.22",
+    "@storybook/ember": "^6.0.22",
+    "@storybook/ember-cli-storybook": "^0.2.1",
+    "babel-eslint": "^10.1.0",
+    "broccoli-asset-rev": "^3.0.0",
+    "ember-auto-import": "^1.6.0",
+    "ember-cli": "~3.21.2",
+    "ember-cli-dependency-checker": "^3.2.0",
+    "ember-cli-inject-live-reload": "^2.0.2",
+    "ember-cli-sri": "^2.1.1",
+    "ember-cli-string-helpers": "^5.0.0",
+    "ember-cli-terser": "^4.0.0",
+    "ember-disable-prototype-extensions": "^1.1.3",
+    "ember-export-application-global": "^2.0.1",
+    "ember-load-initializers": "^2.1.1",
+    "ember-maybe-import-regenerator": "^0.1.6",
+    "ember-qunit": "^4.6.0",
+    "ember-resolver": "^8.0.2",
+    "ember-source": "~3.21.1",
+    "ember-source-channel-url": "^3.0.0",
+    "ember-template-lint": "^2.11.0",
+    "ember-try": "^1.4.0",
+    "eslint": "^7.8.0",
+    "eslint-plugin-ember": "^8.13.0",
+    "eslint-plugin-node": "^11.1.0",
+    "loader.js": "^4.7.0",
+    "npm-run-all": "^4.1.5",
```

Comment:
`npm-run-all` was originally used in generated content for some defined scripts above.  However, those pre-generated scripts have been removed.  We should see if this would be useful for development purposes.